Documentation~/index.md

```
@@ -3,6 +3,47 @@
+
+- **[PLATEAU SDKを使ったGISサンプル](https://www.mlit.go.jp/plateau/learning/tpc17-1/#p17_3_1)**
+![](resources/index/gisexample.jpg)  
+
+- **[PLATEAU SDKを使ったゲームサンプル](https://www.mlit.go.jp/plateau/learning/tpc17-1/#p17_3_2)**
+![](resources/index/gamesamplephoto.png)  
+
+## SDK活用の実践編
+PLATEAU SDK for Unity を用いて実際にコンテンツを構築するチュートリアルをいくつかご紹介します。
+内容は初級・中級・上級の三段階に整理されており、基礎的なカスタマイズから応用的な開発まで、段階的に学習を進められる構成となっています。
+
+#### 初級編：サンプルをカスタマイズして自分のデータに合わせてみよう
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)